### PR TITLE
7z: update Spanish translation

### PR DESCRIPTION
**Pull request by teboarte**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by gutjuri** [comment]
test


For me, the PR adds weird symbols. Are you using UTF-8?

File: pages.es/common/7z.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Lista el contenido de un archivo comprimido:
 
 `7z l {{archivo_comprimido.7z}}`
+
+- Esteblece el nivel de compresión más alto significa más compresión, pero será más lento):
```
Comment (kant) [inline pages.es/common/7z.md:34]: , (pero será más lento):

**Review by kant — CHANGES_REQUESTED** [review]

File: pages.es/common/7z.md (hunk 1 above)
Comment (tricantivu) [inline pages.es/common/7z.md:34]:
```suggestion
- Establece el nivel de compresión más alto (es decir, mayor compresión pero menor velocidad):
```

Comment (tricantivu) [reply pages.es/common/7z.md:34]: What do you think about the suggestion below?

Comment (kant) [reply pages.es/common/7z.md]: Est**a**blece el nivel de compresión más alto significa más compresión, pero será más lento:

**Review by tricantivu — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**